src/modules/GenerateListHTML.js

```diff
@@ -0,0 +1,79 @@
+const list = document.querySelector('.list-ul');
+
+const sortItems = (items) => {
+  const sortedItems = items.sort((a, b) => a.index - b.index);
+  return sortedItems;
+};
```

Comment:
- [ ] You can make this in one line as arrow functions can automatically return the value.

```suggestion
const sortItems = (items) =>  items.sort((a, b) => a.index - b.index);
```